src/main/java/hockeyapp/HockeyappRecorder.java

```diff
@@ -353,17 +339,14 @@ private boolean performForApplication(Run<?, ?> build, FilePath workspace, EnvVa
                 EnvAction envData = new EnvAction();
                 build.addAction(envData);
 
-                if (envData != null) {
```

Comment:
No longer guarding against a null envData? Actually, will it ever not be null?

Reply:
It's safe because Java guarantees that the new operator will never return null.